resotoworker/resotoworker/__main__.py

```diff
@@ -92,6 +94,28 @@ def main() -> None:
         resotocore_uri=resotocore.http_uri,
         tls_data=tls_data,
     )
+
+    # replaces all values wrapped in $() with the environmental valiable
+    def replace_env_vars(elem: JsonElement) -> JsonElement:
+        if isinstance(elem, dict):
+            return {k: replace_env_vars(v) for k, v in elem.items()}
+        elif isinstance(elem, list):
+            return [replace_env_vars(v) for v in elem]
+        elif isinstance(elem, str):
+            str_value = elem
+            for match in re.finditer(r"\$\((\w+)\)", elem):
+                if env_var_found := os.environ.get(match.group(1)):
+                    str_value = str_value.replace(match.group(0), env_var_found)
+
+            return str_value
+        else:
+            return elem
```

Comment:
Shouldn't this code live in resotolib?
I assume it is also relevant for resotometrics etc.
Also resotocore seems to use exactly the same functionality?